components/resources/ResourceList.tsx

```diff
@@ -15,9 +15,12 @@ import {
 import ReviewFilters from '../reviews/review-filters'
 import InfiniteScroll from './InfiniteScrollResources'
 import { useDebounce } from '@/util/hooks/useDebounce'
+import { id } from '@/jest.config'
```

Comment:
What is this being imported for?

Reply:
Whoops. I didn't notice that. That line isn't required.